### ISSUE TITLE
Feature request: add activation zone layer checkbox for Austrian summits using precalculated data

### ISSUE THREAD

**Issue by g84ycm** [manuelkasper/sotlas-frontend#25]
Taking the activation zone layer for Switzerland on sotl.as as an inspiration, I am currently calculating the activation zones for all Austrian SOTA summits based on the official SOTA summit list (https://www.sotadata.org.uk/en/summits) and the official BEV 1m resolution ALS DTM raster data (https://data.bev.gv.at/geonetwork/srv/ger/catalog.search;jsessionid=1F5F6A9D0278E6871FEDB6B87EE0936B#/metadata/eae5f98d-d605-4783-8292-8b913d163cac). For circumventing errors within the SOTA summit list data the official BEV geographical summit names/coordinates are partly used (https://data.bev.gv.at/geonetwork/srv/eng/catalog.search#/metadata/fc46102b-2ced-4518-8cf0-f322229b66db).
The picture below shows the activation zone of OE/VB-070 as an example.

I will publish all the data on Github once the calculation prozess is done (it is calculated state by state). For now, only the activation zones for "Vorarlberg OE9" are finished, see https://github.com/g84ycm/SOTA-Austria-Activation-Zones/tree/main

If this feature is wanted and will be made available on sotl.as in the future, my data can be used freely. If the data format is incompatible with the website, let me know and I will try to convert the data so it complies with your needs.

![activation_zone_oe_vb_070](https://github.com/manuelkasper/sotlas-frontend/assets/55108566/db5ac13b-5e7f-4c43-b3c9-9e48aa7752f2)

**Comment by manuelkasper** [comment]
Awesome! It would be very nice to have OE AZ data on SOTLAS as well. If you could make the polygons available in GeoJSON format (one file per summit, like you already have), that would make it easy for me to import them into the existing vector tile layer.

**Comment by g84ycm** [comment]
Great! I will try to convert it to the GeoJSON format and will let you know when everything is calculated and converted (will probably take some time though).

**Comment by g84ycm** [comment]
Calculations are done. All activation zones can be downloaded in .geojson and .shp file format for each individual summit. The data has been created to the best of my knowledge, but may (and probably will) contain some errors. It is not feasible for me to check all 2100+ activation zones manually. When mistakes are discovered and reported in the future by users of the data, I will updated it accordingly.

All data is provided in the repository mentioned above (https://github.com/g84ycm/SOTA-Austria-Activation-Zones/tree/main).

Additionally, would it be possible to add the "Basemap.at - GRAU" Layer as an additional map option (or replace the currently used basemap.at layer)? In this layer, the contour lines are shown.
![image](https://github.com/manuelkasper/sotlas-frontend/assets/55108566/2553bec4-33e7-4543-959f-b4fc6e974562)

**Comment by manuelkasper** [comment]
Excellent, thanks! This is now live on sotl.as. I have switched to the gray basemap.at version as requested (makes more sense indeed), even if it's not high-res/retina. To make users aware of the possibility to switch to the basemap.at map and thus see the activation zone boundaries, I have also added a one-time popup info dialog that appears for users from Austrian IP addresses.

**Comment by g84ycm** [comment]
Looks very nice, thank you for implementing it!
People from the OE SOTA community have asked if it would be possible to make the layer available not only while using the basemap.at map type, but also when using the standard map type?

**Comment by manuelkasper** [comment]
> People from the OE SOTA community have asked if it would be possible to make the layer available not only while using the basemap.at map type, but also when using the standard map type?

Yes, why not, seeing as we now have two (well, two and a half with HB0 😉) associations with association zone data. Done. Hopefully activation zone polygons for other associations will be contributed in the future.

**Comment by g84ycm** [comment]
Great, thank you!
